src/main/java/org/variantsync/diffdetective/feature/PropositionalFormulaParser.java

```diff
@@ -36,6 +37,11 @@ public interface PropositionalFormulaParser {
             node = FixTrueFalse.EliminateTrueAndFalseInplace(node).get();
         }
 
+        if (node == null) {
+//            Logger.warn("Could not parse expression '{}' to feature mapping. Using it as literal.", fmString);
+            node = new Literal(text);
+        }
```

Comment:
This looks like an important fix? Was this code moved? If so, where did it came from?

Reply:
This code was originally part of CPPAnnotationParser and always executed after PropositionalFormulaParser was called. So I moved it inside the PropositionalFormulaParser. There are no other callers affected by this change.

Reply:
Ok good idea!